content/reference/monitoring.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Monitoring and Observability"
+weight: 3
+---
+
+Monitoring your libp2p process lets make sure that in practice things are going
+as you expect. Each libp2p implementation does monitoring in a slightly
+different way, so consult the links below to see how your implementation does
+it.
+
+Rust libp2p:
+
+Look at the [libp2p-metrics crate](https://github.com/libp2p/rust-libp2p/tree/master/misc/metrics).
+
+Go libp2p:
+
+For resource usage take a look at the OpenCensus metrics exposed by the resource
+manager
+[here](https://pkg.go.dev/github.com/libp2p/go-libp2p-resource-manager@v0.5.2/obs).
```

Comment:
Is there a way to link to @latest somehow?

Reply:
Generally a preference for a link that will always work, but not a strong opinion here. I'll change it to point to latest.